load-testing/audit-service/sendLoginMessages.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Populates the SQS queue with couple of 100 messages for audit service load testing
+#
+
+export AWS_DEFAULT_REGION=us-east-1
+
+message= '
+{
+    "idp":"fence",
+    "request_url": "",
+    "status_code": 200,
+    "sub": 1,
+    "username": "user_loadtest",
+}'
+
+# send 20 dummy logs to audit-service sqs
+for i in {1..20};
+    do
+        aws sqs send-message --queue-url "https://sqs.us-east-1.amazonaws.com/707767160287/${auditSQS}" --message-body "$message"
```

Comment:
It would be nice to parameterize the name of the queue through an environment variable to make the script more flexible (easy to use).